cranelift-codegen/src/legalizer/mod.rs

```diff
@@ -124,7 +124,6 @@ pub fn legalize_function(func: &mut ir::Function, cfg: &mut ControlFlowGraph, is
 include!(concat!(env!("OUT_DIR"), "/legalizer.rs"));
 
 /// Custom expansion for conditional trap instructions.
-/// TODO: Add CFG support to the Python patterns so we won't have to do this.
```

Comment:
Please leave this comment, we could still add CFG support to the Rust patterns later.